api-specificatie/openapi.yaml

```diff
@@ -1054,519 +1003,215 @@ components:
           $ref: '#/components/schemas/NEN3610ID'
         adresseerbaarObjectIdentificatie:
           $ref: '#/components/schemas/NEN3610ID'
-        pandIdentificatie:
+        pandIdentificaties:
           type: array
           items:
             $ref: '#/components/schemas/NEN3610ID'
           minItems: 1
-        nevenadres:
-          description: 'Boolean die aangeeft of dit adres een nevenadres is. Als deze boolean niet wordt meegeleverd (of als deze false is) dan is dit een Hoofdadres.'
+        indicatieNevenadres:
+          description: 'Een aanduiding die aangeeft of dit adres een nevenadres is van een adresseerbaar object. Als deze boolean niet wordt meegeleverd (of als deze false is) dan is dit een Hoofdadres.'
           type: boolean
-        geconstateerd:
-          description: 'Een aanduiding waarmee kan worden aangegeven dat een object in de registratie is opgenomen als gevolg van een feitelijke constatering, zonder dat er op het moment van opname sprake was van een regulier brondocument voor deze opname.'
+        indicatieGeconstateerd:
+          description: 'Een aanduiding waarmee kan worden aangegeven dat één of meerdere gerelateerde object(en) in de registratie is/zijn opgenomen als gevolg van een feitelijke constatering, zonder dat er op het moment van opname sprake was van een regulier brondocument voor deze opname.'
           type: boolean
-        indicatieInOnderzoek:
-          description: 'Een indicatie dat één of meerdere attributen van objecten gerelateerd aan een adres in onderzoek is/zijn.'
+        indicatieMogelijkOnjuist:
+          description: 'Een aanduiding dat één of meerdere attributen van objecten gerelateerd aan een adres in onderzoek of mogelijk onjuist is/zijn.'
           type: boolean
-
+    AdresHal:
+      type: object
+      allOf:
+      - $ref: '#/components/schemas/AdresHalBasis'
+      - properties:
+          _embedded:
+            $ref: '#/components/schemas/Adres_embedded'
+    AdresHalBasis:
+      type: object
```

Comment:
'type: object' geeft hier geen foutmelding, maar hoort hier niet. Volgens mij moet het zijn:

      allOf:
      - $ref: '#/components/schemas/Adres'
      - object:
        properties:
          _links:
             $ref: '#/components/schemas/Adres_links'

Zie: Models with Composition @ https://swagger.io/specification/

Andere regels waar dit ook zo is gespecificeerd:
- 1130
- 1138
- 1267
- 1274
- 1366
- 1373
- 1448
- 1453
- 1517

Reply:
In het geschetste voorbeeld mij "Models with Composition"  staat er "- type: object"  ipv "- object" . Dat hebben we in het verleden eerder toegepast en toen ontstonden er volgens mij code-generatie-problemen. Ik heb de structuur nu aangepast conform de BRP. Dus : 
   allOf:
  - $ref: '#/components/schemas/Adres'
  - properties:
      _links:
         $ref: '#/components/schemas/Adres_links'

Maar als we hier beter "- type: object voor kunnen zetten doe ik dat. Gaarne morgen nog even bespreken.